go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/decred/dcrd/txscript/v2 v2.1.0
 	github.com/decred/dcrd/wire v1.3.0
 	github.com/decred/dcrwallet/rpc/jsonrpc/types v1.4.0
-	github.com/decred/dcrwallet/wallet/v3 v3.1.0
+	github.com/decred/dcrwallet/wallet/v3 v3.1.1-0.20191230143837-6a86dc4676f0
```

Comment:
Could use a tidy to purge the previous wallet/v3 checksums, but it's not important at this point.